client/src/components/desk/pickup/ReadyForPickupList.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {Container, Header, Segment} from "semantic-ui-react";
+import CardList from "../CardList";
+import ReadyForPickupCard from "./ReadyForPickupCard";
+
+function ReadyForPickupList({cards}: any) {
+    const empty = <Segment placeholder>
```

Comment:
do we not need parens anymore

Reply:
IntelliJ and React are fine with it, but it looks like you should add the parentheses around multiline JSX statements to prevent JavaScript from adding semicolons in the wrong place.  I'll go through and fix that.

https://medium.com/@leannezhang/curly-braces-versus-parenthesis-in-reactjs-4d3ffd33128f